third_party/llvm-project/include/llvm/ObjectYAML/DWARFYAML.h

```diff
@@ -82,6 +82,7 @@ struct Loc {
   uint32_t Start;
   uint32_t End;
   std::vector<uint8_t> Location;
+  uint64_t Offset;
```

Comment:
Perhaps add a comment saying what offset this is? Or the name could mention that it is the compile unit offset?